packages/contracts/contracts/QuadraticVoteCollectModule.sol

```diff
@@ -94,7 +68,6 @@ contract QuadraticVoteCollectModule is FeeModuleBase, FollowValidationModuleBase
         uint256 pubId,
         bytes calldata data
     ) external {
-        if (_dataByPublicationByProfile[profileId][pubId].followerOnly) _checkFollowValidity(profileId, collector);
```

Comment:
why remove this?  won't this help to keep all the original lens collect features?

Reply:
Do we only allow followers to vote?